presto-main/src/main/java/com/facebook/presto/operator/BigintGroupByHash.java

```diff
@@ -303,9 +303,9 @@ private void rehash()
         this.valuesByGroupId.ensureCapacity(maxFill);
     }
 
-    private static int getHashPosition(long rawHash, int mask)
+    private static long getHashPosition(long rawHash, int mask)
     {
-        return ((int) murmurHash3(rawHash)) & mask;
+        return murmurHash3(rawHash) & mask;
```

Comment:
That's interesting. Murmur3 is either a 32 bit or a128 bit hash. I've no idea what hash that is :)

It's ok to leave as is, then. Eventually, we should switch to xxhash64 for consistency, anyway.